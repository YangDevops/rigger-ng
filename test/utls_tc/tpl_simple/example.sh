echo "this is rg tpl"
echo "Hi,%{YOUNAME}"
echo "love rg?"
#% T.LOVE : {
echo "love you too! "
#% }
#% T.DB == "mysql" : {
echo "mysql is free! "
#% }
#% T.DB == "orcal" : {
echo "you are tuhao!"
#% }
