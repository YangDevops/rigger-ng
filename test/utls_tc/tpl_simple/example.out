echo "this is rg tpl"
echo "Hi,abc"
echo "love rg?"
